### ISSUE TITLE
Upgrade Tensorflow to the latest version 2.4 to 2.6

### ISSUE THREAD

**Issue by pouria-faraji** [Kaggle/docker-python#1064]
Tensorflow is a little bit outdated. For example, I wanted to use the AUC metric with the "from_logits" argument, I realized that in version 2.4 it does not exist.

**Comment by rosbo** [comment]
Hi @pouria-faraji,

We are actively working on upgrading to the latest version of TensorFlow but we were hitting some compatibility issues with the other packages included in the image.

Hopefully, we will be able to complete the upgrade and release it to Kaggle Kernels in the coming weeks.

Thank you for raising the issue.

**Comment by rosbo** [comment]
The next Docker image release in 2 weeks will include TensorFlow 2.6
